# Diff: https://project-skyscraper.com/wp-json/wp/v2/search?per_page=100
# File: api/wp-json/wp/v2/search_per_page_100.json
# Timestamp: 2026-05-31 06:30:32 UTC
# Lines: 1352 old -> 1442 new
# Beautified: yes
--- old/api/wp-json/wp/v2/search_per_page_100.json
+++ new/api/wp-json/wp/v2/search_per_page_100.json
@@ -1,4 +1,94 @@
 [
+  {
+    "id": 288,
+    "title": "Security Log",
+    "url": "https://project-skyscraper.com/sec-log-094316-full/",
+    "type": "post",
+    "subtype": "page",
+    "_links": {
+      "self": [
+        {
+          "embeddable": true,
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/pages/288",
+          "targetHints": {
+            "allow": [
+              "GET"
+            ]
+          }
+        }
+      ],
+      "about": [
+        {
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/types/page"
+        }
+      ],
+      "collection": [
+        {
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/search"
+        }
+      ]
+    }
+  },
+  {
+    "id": 291,
+    "title": "SECURITY LOG",
+    "url": "https://project-skyscraper.com/2026/05/30/sec-log-memory-sector-access/",
+    "type": "post",
+    "subtype": "post",
+    "_links": {
+      "self": [
+        {
+          "embeddable": true,
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/posts/291",
+          "targetHints": {
+            "allow": [
+              "GET"
+            ]
+          }
+        }
+      ],
+      "about": [
+        {
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/types/post"
+        }
+      ],
+      "collection": [
+        {
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/search"
+        }
+      ]
+    }
+  },
+  {
+    "id": 1776,
+    "title": "Request Memory Timestamp",
+    "url": "https://project-skyscraper.com/request-memory-timestamp-094317/",
+    "type": "post",
+    "subtype": "page",
+    "_links": {
+      "self": [
+        {
+          "embeddable": true,
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/pages/1776",
+          "targetHints": {
+            "allow": [
+              "GET"
+            ]
+          }
+        }
+      ],
+      "about": [
+        {
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/types/page"
+        }
+      ],
+      "collection": [
+        {
+          "href": "https://project-skyscraper.com/wp-json/wp/v2/search"
+        }
+      ]
+    }
+  },
   {
     "id": 1833,
     "title": "SECURITY LOG",
@@ -722,7 +812,7 @@
   {
     "id": 551,
     "title": "Neural network status",
-    "url": "https://project-skyscraper.com/",
+    "url": "https://project-skyscraper.com/neural-network-status/",
     "type": "post",
     "subtype": "page",
     "_links": {
